packages/database/src/migrations/20220823234713-AddLesmisFavouriteDashboardTable-modifies-schema.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
```

Comment:
Could rename this file to: `20220823234713-AddUserFavouriteDashboardTable-modifies-schema.js`

No biggie though 👍